Build warning Problem

**Describe the bug**
Type mismatch in `FindPayloadInRemoteDetourSection`:
```
~/detours/creatwth.cpp: In function ‘void* FindPayloadInRemoteDetourSection(HANDLE, const GUID&, DWORD*, PVOID)’:
~/detours/creatwth.cpp:194:20: warning: converting to non-pointer type ‘DWORD’ {aka ‘long unsigned int’} from NULL [-Wconversion-null]
  194 |         *pcbData = NULL;
      |                    ^~~~
```

Should the assignment be to 0 since `*pcbData`  is not a pointer type?

**Expected behavior**
Builds without warnings

**Detours version**
https://github.com/microsoft/Detours/commit/827b89608ce66892963d1467d15a86f0b2832a78

--- a/src/creatwth.cpp
+++ b/src/creatwth.cpp
@@ -191,7 +191,7 @@ static PVOID FindPayloadInRemoteDetourSection(_In_ HANDLE hProcess,
                                                _In_ PVOID pvRemoteDetoursSection)
 {
     if (pcbData) {
-        *pcbData = NULL;
+        *pcbData = 0;
     }
 
     PBYTE pbData = (PBYTE)pvRemoteDetoursSection;

